presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -1500,6 +1501,19 @@ public double getDefaultJoinSelectivityCoefficient()
         return defaultJoinSelectivityCoefficient;
     }
 
+    @Config("optimizer.default-cte-producer-replication-coefficient")
+    @ConfigDescription("The replication coefficient for CTE producers which aids the cost calculator in determining the multiplier that quantifies the expense of writing operations")
+    public FeaturesConfig setDefaultCteProducerReplicationCoefficient(double defaultJoinSelectivityCoefficient)
```

Comment:
also perhaps a simpler name? something like defaultCteReplicationCoefficient? or defaultCteMaterializationCoefficient?

Reply:
do we have other parameters that are specific to writing costs? this isn't necessarily specific to CTEs, but any writes that are done by the database. Just wondering if we should go more generic with the name or keep it specific to CTEs
anwyays, just a nit

Reply:
Hmm, the cost calculator doesn't have an estimation for writes and I don't see any other parameters, makes sense to use a generic name